### PR TITLE
Lpd 19909

### PR DESCRIPTION
**Pull request by brianikim**
https://liferay.atlassian.net/browse/LPD-19909,

@smottal , tied to LPP.

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:playwright

File: modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/servlet/taglib/util/UserActionDropdownItems.java [hunk 1]
```diff
@@ -107,6 +108,8 @@ public List<DropdownItem> getActionDropdownItems() {
 						PropsValues.PORTAL_IMPERSONATION_ENABLE &&
 						(_user.getUserId() != _themeDisplay.getUserId()) &&
 						_user.isActive() && !_themeDisplay.isImpersonated() &&
+						!Objects.equals(
```
Comment (smottal) [inline modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/servlet/taglib/util/UserActionDropdownItems.java:111]: Can you move this after PropsValues.PORTAL_IMPERSONATION_ENABLE but before (_user.getUserId() != _themeDisplay.getUserId()) ? Thx

**Review by smottal — CHANGES_REQUESTED** [review]
@brianikim only a very little change on this. Thx

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:playwright

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:stable

**Comment by brianikim** [comment]
ci:test:playwright

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:forward

**Pull request closed without merging**